apps/api/src/domain/models/audio-visual/audio-item/queries/audio-item-query-repository.interface.ts

```diff
@@ -29,6 +29,7 @@ export interface IAudioItemQueryRepository
         IQueryRepositoryForTaggable,
         IQueryRepositoryForAnnotatable,
         IQueryRepositoryForConnectable {
+    // IQueryRepositoryForAttributable
```

Comment:
I'll keep an eye on this one. Maybe I'll make sure this is there on my current branch.

Reply:
This has been done on a different branch.